client/src/components/ModalDialog/ModalDialog.module.css

```diff
@@ -0,0 +1,44 @@
+.modalWrapper.displayed {
+  display: block;
+}
+
+.modalWrapper.hidden {
+  display: none;
+}
+
+.scrim {
+  position: fixed;
+  z-index: 9999;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  pointer-events: all;
+  background: rgba(0, 0, 0, 0.25);
+}
+
+.modal {
+  background: white;
+  position: absolute;
+  top: 45%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  padding: 40px 80px;
+  border-radius: 10px;
+  box-shadow: 0 10px 40px -20px rgba(0, 0, 0, 0.25);
```

Comment:
```suggestion
  box-shadow: 0px 1px 4px 1px rgba(150,150,150,0.75);
```

Reply:
where the styles you're suggesting come from?

Reply:
The screenshot in #74 looks like it has a teeny drop shadow